src/routes/view-content/[guideId]/[bibleSection]/library-resource-menu/LibraryResourceMenu.svelte

```diff
@@ -157,7 +157,7 @@
         if (!isShowing || (!isFullLibrary && resourceGroupings?.length > 0)) return;
         isLoading = true;
 
-        if ($isPassageSearch) {
+        if ($isPassageSearch && $currentTab === ContentTabEnum.LibraryMenu) {
```

Comment:
Nit: I wonder if there could be a new reactive variable above to simplify some of this like:
```
$: showingPassageSearch = $isPassageSearch && isFullLibrary
```
The `isFullLibrary` feels like a more direct way of determining if it's the Library tab.